lib/plugins/issues.js

```diff
@@ -54,4 +54,13 @@ module.exports = class Issues extends Plugin {
 
     return deleteFunction(context.toRepo({id: comment.id}));
   }
+
+  create(context, content) {
+    var lines = content.split('\n');
```

Comment:
> This may be better if refactored to take two arguments in configuration: one for the Title and one for the Body. I had originally tried to pass these in as a JSON object, but I was having trouble getting the body content to resolve properly with handlebars. Pending further conversation, I think I'd like to revisit this approach.

Yeah, I think doing an object would be ideas to allow for all the other attributes that the issues API accepts (`title`, `body`, `assignees`, `labels`). I can help debug issues with passing content.